File: src/java/com/distributedlife/mahjong/reference/permute/PairPermutator.java
package com.distributedlife.mahjong.reference.permute;

import java.util.List;

public class PairPermutator extends MultiplePermutator {
    public PairPermutator(List<String> tilesPairIsAllowedIn, String suit) {
        super(tilesPairIsAllowedIn, 2, suit);
    }
}

File: lib/permute/pair-permutator.js
/**
 * PairPermutator - Handles permutations for pairs (2 tiles)
 */
import { MultiplePermutator } from './multiple-permutator.js';

export class PairPermutator extends MultiplePermutator {
  constructor(tilesPairIsAllowedIn, suit) {
    super(tilesPairIsAllowedIn, 2, suit);
  }
}
